lib/core/network/api_service.dart

```diff
@@ -234,4 +234,51 @@ class APIService {
       return itemsList;
     }
   }
+
+  Future<List<dynamic>> fetchAvatars() async {
+    final url = Uri.https(_apiURL, '/givt4kidsservice/v1/profiles/avatars');
+
+    final response = await client.get(url);
+
+    log('fetch avatars response code: ${response.statusCode}');
```

Comment:
I would remove the debug logging from the code. (its getting logged to logit in dev and prod)